ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ranger/RangerRestClientImpl.java

```diff
@@ -202,12 +238,14 @@ public RangerExportPolicyList importRangerPolicies(RangerExportPolicyList ranger
     String finalUrl = getRangerImportUrl(baseUrl, dbName);
     LOG.debug("URL to import policies on target Ranger: {}", finalUrl);
     Retryable retryable = Retryable.builder()
-      .withHiveConf(hiveConf)
-      .withRetryOnException(Exception.class).build();
+      .withHiveConf(hiveConf).withFailOnException(RuntimeException.class)
+      .withRetryOnException(SemanticException.class).build();
     try {
       return retryable.executeCallable(() -> importRangerPoliciesPlain(jsonRangerExportPolicyList,
               rangerPoliciesJsonFileName,
               serviceMapJsonFileName, jsonServiceMap, finalUrl, rangerExportPolicyList, hiveConf));
+    } catch (RuntimeException e) {
```

Comment:
you are catching a runtime exception and throwing a semantic exception. why are you doing that?